libs/docusaurus/src/schematics/app/schematic.spec.ts

```diff
@@ -19,22 +21,180 @@ describe('docusaurus schematic', () => {
     appTree.create('.prettierignore', '');
   });
 
+  it('should update workspace.json', async () => {
+    const tree = await testRunner
+      .runSchematicAsync('app', options, appTree)
+      .toPromise();
+
+    const workspaceJson = readJsonInTree(tree, 'workspace.json');
+    const { build, serve } = workspaceJson.projects['my-app'].architect;
+
+    expect(workspaceJson.projects['my-app'].root).toBe('apps/my-app');
+    expect(build.builder).toBe('@nx-plus/docusaurus:browser');
+    expect(build.options).toEqual({ outputPath: 'dist/apps/my-app' });
+    expect(serve.builder).toBe('@nx-plus/docusaurus:dev-server');
+    expect(serve.options).toEqual({
+      port: 3000,
+    });
+  });
+
+  it('should generate files', async () => {
+    const tree = await testRunner
+      .runSchematicAsync('app', options, appTree)
+      .toPromise();
+
+    [
+      'apps/my-app/docusaurus.config.js',
+      'apps/my-app/docs/doc3.md',
+      'apps/my-app/docs/doc2.md',
+      'apps/my-app/docs/doc1.md',
+      'apps/my-app/docs/mdx.md',
+      'apps/my-app/blog/2019-05-29-hello-world.md',
+      'apps/my-app/blog/2019-05-28-hola.md',
+      'apps/my-app/blog/2019-05-30-welcome.md',
+      'apps/my-app/static/img/undraw_docusaurus_mountain.svg',
+      'apps/my-app/static/img/logo.svg',
+      'apps/my-app/static/img/favicon.ico',
+      'apps/my-app/static/img/undraw_docusaurus_react.svg',
+      'apps/my-app/static/img/undraw_docusaurus_tree.svg',
+      'apps/my-app/sidebars.js',
+      'apps/my-app/babel.config.js',
+      'apps/my-app/src/css/custom.css',
+      'apps/my-app/src/pages/index.js',
+      'apps/my-app/src/pages/styles.module.css',
+    ].forEach((path) => expect(tree.exists(path)).toBeTruthy());
+
+    expect(tree.readContent('.gitignore')).toContain(tags.stripIndent`
+      # Generated Docusaurus files
+      .docusaurus/
+      .cache-loader/
+    `);
+
+    expect(tree.readContent('.prettierignore')).toContain('.docusaurus/');
+  });
+
   describe('--directory', () => {
-    it('should create src in the specified directory', async () => {
+    it('should update workspace.json', async () => {
       const tree = await testRunner
         .runSchematicAsync('app', { ...options, directory: 'subdir' }, appTree)
         .toPromise();
-      expect(tree.exists('apps/subdir/test/docusaurus.config.js')).toBeTruthy();
+
+      const workspaceJson = readJsonInTree(tree, 'workspace.json');
+      const { build, serve } = workspaceJson.projects[
+        'subdir-my-app'
+      ].architect;
+
+      expect(workspaceJson.projects['subdir-my-app'].root).toBe(
+        'apps/subdir/my-app'
+      );
+      expect(build.builder).toBe('@nx-plus/docusaurus:browser');
+      expect(build.options).toEqual({ outputPath: 'dist/apps/subdir/my-app' });
+      expect(serve.builder).toBe('@nx-plus/docusaurus:dev-server');
+      expect(serve.options).toEqual({
+        port: 3000,
+      });
```

Comment:
Remove the assertions in this block except for the following:

```
expect(build.options).toEqual({ outputPath: 'dist/apps/subdir/my-app' });
```

I don't think `--directory` requires making the other assertions again. Same for the test below.